integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/standardpartition/StandardPartitionTableLoadingTestCase.scala

```diff
@@ -28,7 +31,7 @@ import org.apache.carbondata.core.util.CarbonProperties
 import org.apache.carbondata.core.util.path.CarbonTablePath
 
 class StandardPartitionTableLoadingTestCase extends QueryTest with BeforeAndAfterAll {
-
+  var executorService: ExecutorService = _
```

Comment:
move to within the testcase as only that testcase uses it

Reply:
yes, but in case that testcase fails also i want to shutdown in afterall thats why added in class level